src/pages/billing/List.js

```diff
@@ -362,9 +362,10 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
           ])],
         [selectedName && hasBillingProjects, () => h(ProjectDetail, {
           key: selectedName,
-          project: _.find({ projectName: selectedName }, billingProjects),
+          billingProject: _.find({ projectName: selectedName }, billingProjects),
           billingAccounts,
-          authorizeAndLoadAccounts
+          authorizeAndLoadAccounts,
+          invalidateProjectAndReload: loadProjects
```

Comment:
slightly confused about the naming here, why/how does this invalidate the project?

Reply:
I was thinking along the lines of marking an element in a cache as invalidand triggering a reload. Really what I'm trying to convey is that the whole component needs to be reloaded with an updated project. maybe `updateProjectAndReload` is more suitable.